src/PrestaShopBundle/Controller/Admin/Sell/Catalog/FeatureValueController.php

```diff
@@ -322,6 +324,36 @@ public function getFeatureValuesAction(
         return new JsonResponse($data);
     }
 
+    /**
+     * Changes feature value position
+     *
+     * @param Request $request
+     *
+     * @return Response
+     */
+    #[AdminSecurity("is_granted('update', request.get('_legacy_controller'))")]
+    public function updatePositionAction(
```

Comment:
You don't seem shaken at all by the new `PrestaShopAdminController` even without the doc being updated yet, great job!